### ISSUE TITLE
Why does it have the js/jquery.js

### ISSUE THREAD

**Issue by mudrd8mz** [ethz-let/qtype_scmc#8]
Moodle already has inbuilt support for jQuery and plugins are not normally expected to ship with their own jQuery version. Is there some reason for this?

**Comment by nexterday** [comment]
The jQuery is to support old versions of moodle (especially with custom old themes which have no jquery). for the new moodle versions, we will not include the jQuery.

**Comment by mudrd8mz** [comment]
The plugin declare it requires Moodle >= 3.4 (2017111300). JQuery was added as an AMD module to the Moodle core in its version 2.9. It has nothing to do with actually used theme as far as I know.

**Comment by nexterday** [comment]
Our own theme we use for day-to-day is running since 2.4 and contains no jQuery.

**Comment by mudrd8mz** [comment]
Yup, I understand that and I can see why the jQuery was needed before Moodle 2.9. I just believe it is no longer needed in later than that versions. This issue has been reported as a well meant reminder / todo item to remove the custom jQuery version from the plugin.
